src/server/scripts/Kalimdor/BlackfathomDeeps/blackfathom_deeps.cpp

```diff
@@ -238,10 +239,41 @@ class npc_morridune : public CreatureScript
     }
 };
 
+class spell_subjugator_korul_darkness_calls : public SpellScriptLoader  // 151159
+{
+    public:
+        spell_subjugator_korul_darkness_calls() : SpellScriptLoader("spell_subjugator_korul_darkness_calls") { }
+
+        class spell_subjugator_korul_darkness_calls_SpellScript : public SpellScript
+        {
+            PrepareSpellScript(spell_subjugator_korul_darkness_calls_SpellScript);
+
+            void HandleScript(SpellEffIndex /*effIndex*/)
+            {
+                if (Creature* caster = GetCaster()->ToCreature())
+                {
+                    caster->CastSpell(target, uint32(GetEffectValue()))
```

Comment:
caster->CastSpell(GetHitUnit(), uint32(GetEffectValue()))

try this, better working it out here than closing the pr again, else #15316 will be open for another 2 years